user_service/app/views.py

```diff
@@ -0,0 +1,262 @@
+from aiohttp import web
+from models import User, users
+from aiohttp_session import get_session, new_session
+import jwt
+from datetime import timedelta, datetime
+import sqlalchemy as sa
+import secrets
+import math
+from settings import JWT
+import tokens
+
+SECRET_KEY = JWT["SECRET"]
+ALGORITHM = JWT["ALGORITHM"]
+ACCESS_TOKEN_EXPIRE_MINUTES = 30
+REFRESH_TOKEN_EXPIRE_DAYS = 30
+
+
+# registration of new user with 4 params(username, password, name, age)
+async def signin(request):
+    """
+    ---
+    description: This route allow to register users
+    tags:
+    - Registration
+    produces:
+    - text/plain
+    parameters_strategy: merge
+    omit_parameters:
+        - path
+    parameters:
+        - name: Username
+          description: Enter your username
+          required: true
+          type: string
+          paramType: form
+        - name: Password
+          description: Enter your password
+          required: true
+          type: string
+          paramType: form
+        - name: Name
+          description: Enter your name
+          required: true
+          type: string
+          paramType: form
+        - name: Age
+          description: Enter your age
+          required: true
+          type: string
+          paramType: form
+    """
+    data = [
+        request.rel_url.query["username"],
+        request.rel_url.query["password"],
+        request.rel_url.query["name"],
+        request.rel_url.query["age"],
+    ]
+
+    user = User(request.app["db"], data)
+    # creating a new user
+    result = await user.create_user()
+    if result:
+        return web.Response(
+            content_type="application/json",
+            text=tokens.convert_json_status("Success")
+        )
+    else:
+        return web.Response(
+            content_type="application/json",
+            text=tokens.convert_json_status("User exists"),
+        )
```

Comment:
You can replace ```web.Response()```,  with ```web.json_response()```, it will be more compact and relevant. 
doc about this method: https://docs.aiohttp.org/en/stable/web_reference.html#json-response

```suggestion
        return web.json_response(text=tokens.convert_json_status("User exists"))
```

Reply:
i wrote a special function to convert to json, so i don’t need in ```web.json_response```

Reply:
если сделать как ты написал то будет преобразование из json в json и получится непонятное преобразование(а вообще зачем ты на английском написал, я же свой))

Reply:
да, забыл. web.json_response(text="User exists")
так же лучше, нет?
идк, по рофлу

Reply:
я хотел сделать кастомный json ответ, поэтому я написал спецом функцию для красивого json, можешь чекнуть ```tokens.py```

Reply:
**_понял_**

Reply:
Найс батл, ребята. json_response рил удобнее.